tbdex/rfq/rfq.go

```diff
@@ -60,10 +61,14 @@ func (r *RFQ) Verify(privateDataStrict bool) error {
 		return fmt.Errorf("failed to verify RFQ signature: %w", err)
 	}
 
-	// TODO add check when decoded.SignerDID is implemented
-	// if decoded.SignerDID != r.MessageMetadata.From {
-	// 	return errors.New("signer: %w does not match message metadata from: %w", decoded.Header.SignerDID, r.MessageMetadata.From)
-	// }
+	decoded, err := jws.Decode(r.Signature)
```

Comment:
Don't need to call this again, `tbdex.VerifySignature` returns the decoded jws so you can just check against that directly by updating ln 59 to:

```
decoded, err := tbdex.VerifySignature(r, r.Signature)
```